---
title             : "Using environmental DNA (eDNA) to identify suitable habitat for freshwater mussels in the James River watershed"
shorttitle        : "Title"

author: 
  - name          : "Nicholas Duellman"
    affiliation   : "1"
    corresponding : yes    # Define only one corresponding author
    address       : "Postal address"
    email         : "my@email.com"
    role: # Contributorship roles (e.g., CRediT, https://credit.niso.org/)
      - "Conceptualization"
      - "Writing - Original Draft Preparation"
      - "Writing - Review & Editing"
  - name          : ""
    affiliation   : ""
    role:
      - "Writing - Review & Editing"
      - "Supervision"

affiliation:
  - id            : "1"
    institution   : "Longwood University"
  - id            : "2"
    institution   : "Cook - Cole College of Science"

authornote: |
  Add complete departmental affiliations for each author here. Each new line herein must be indented, like this line.

  Enter author note here.

abstract: |
  One or two sentences providing a **basic introduction** to the field,  comprehensible to a scientist in any discipline.
  Two to three sentences of **more detailed background**, comprehensible  to scientists in related disciplines.
  One sentence clearly stating the **general problem** being addressed by  this particular study.
  One sentence summarizing the main result (with the words "**here we show**" or their equivalent).
  Two or three sentences explaining what the **main result** reveals in direct comparison to what was thought to be the case previously, or how the  main result adds to previous knowledge.
  One or two sentences to put the results into a more **general context**.
  Two or three sentences to provide a **broader perspective**, readily comprehensible to a scientist in any discipline.
  
  <!-- https://tinyurl.com/ybremelq -->
  
keywords          : "keywords"
wordcount         : "X"

bibliography      : "r-references.bib"

floatsintext      : no
linenumbers       : yes
draft             : no
mask              : no

figurelist        : no
tablelist         : no
footnotelist      : no

classoption       : "man"
output            : papaja::apa6_pdf
---

```{r setup, include = FALSE}
library("papaja")
r_refs("r-references.bib")
```

```{r analysis-preferences}
# Seed for random number generation
set.seed(42)
knitr::opts_chunk$set(cache.extra = knitr::rand_seed)
```
# Introduction

Freshwater mussels play a key role in freshwater ecosystems, specifically positively impacting denitrification and influencing nutrient cycling. Mussel populations also contribute to river composition resilience and mitigate substrate erosion, providing further resilience to riparian buffers (Spooner and Vaughn 2006, Vaughn 2018). However, the extent of available mussel habitat is currently unresolved in the Appomattox and James River. In particular, the James SpinyMussel is endemic in this watershed, and its habitat is completely unresolved and it is considered endangered under the Endangered Species Act. In addition, current traditional field sampling methods are notoriously laborious and financially inefficient. 
The analysis of environmental DNA (eDNA) is increasing in relevance within the field of conservation biology due to cost effectiveness and efficiency for data acquisition. Utilization of eDNA for field sampling methods requires far less fieldwork and cost over traditional field sampling methods by providing efficient sampling and data collection. Target mussel eDNA data will be used to identify and replicate suitable habitats, both improving water quality and promoting riparian buffer restoration. 
MinION (Oxford Nanopore Technologies) can detect different genera within the same eDNA samples. MinION provides real time sequencing directly from DNA molecules, reducing the need for reverse transcription. The dada2 (Callahan et al., 2016) R package allows for downstream analysis of DNA data. Dada2 normalizes and trims the fastq data imported from MinION sequencing. 
	However, DNA-seq data still requires a data pipeline to further identify species of freshwater mussels for detection. Data quality assessment, trimming and interpretation with a genomic pipeline for freshwater mussels proved to be a discrepancy with MinION and dada2. To combat this, I developed a single-read DNA pipeline using the dada2 package (Callahan et al., 2016). The work was adapted from Steyaert et al. 2020 approach to metabarcoding with R. Additionally, I designed 16s primers for Elliptio, Fusconia, Pyganodon genus and a novel primer for the Parvaspina genus. The James SpinyMussel is a critically endangered endemic species to North Carolina and Virginia, so the development of a primer for the proposed monitoring of the species is critical within eDNA research. Our team wants to utilize the genomic pipeline to further identify a total of 14 endemic species, all at least IUCN listed vulnerable. 



# Methods

We report how we determined our sample size, all data exclusions (if any), all manipulations, and all measures in the study. <!-- 21-word solution (Simmons, Nelson & Simonsohn, 2012; retrieved from http://ssrn.com/abstract=2160588) -->

## Participants

## Material

## Procedure

## Data analysis

We used `r cite_r("r-references.bib")` for all our analyses.

# Results

# Discussion

\newpage

# References

::: {#refs custom-style="Bibliography"}
:::
